docs/references/references.bib

```diff
@@ -395,3 +395,23 @@ @misc{Adcock:2021
   howpublished = {APS DFD},
   year = {2021},
 }
+
+@article{chien1982predictions,
+  title={Predictions of channel and boundary-layer flows with a low-Reynolds-number turbulence model},
+  author={Chien, Kuei-Yuan},
+  journal={AIAA journal},
+  volume={20},
+  number={1},
+  pages={33--38},
+  year={1982}
+}
+
+@article{wilcox2008formulation,
```

Comment:
I think I implemented `Wilcox, D. C., Turbulence Modeling for CFD, 2nd edition, DCW Industries, Inc., La Canada CA, 1998` but let me double check

Reply:
Ok lmk

Reply:
fixed